compute_tools/src/pg_helpers.rs

```diff
@@ -44,7 +44,7 @@ pub fn escape_conf_value(s: &str) -> String {
     format!("'{}'", res)
 }
 
-trait GenericOptionExt {
+pub trait GenericOptionExt {
```

Comment:
I think this can be made `pub(crate)`. I'm not a Rust person so take it with a grain of salt :).

Reply:
Fair enough :) I was copying how `PgOptionsSerialize` is public but tbh I don't really know why it is like that currently... I see it was added _here_ in f64074c60983a38dc692ef4dd89ad5d69202b3ec, which itself was just copied from the original implementation where the trait was made public in neondatabase/cloud#46 (!!)

Reply:
Haha fair enough, thanks :)